dockers/docker-router-advertiser/radvd.conf.j2

```diff
@@ -0,0 +1,37 @@
+{% block banner %}
+# =========== Managed by sonic-cfggen -- DO NOT edit manually! ====================
+# Generated by /usr/share/sonic/templates/radvd.conf.j2 using config DB data
+# File: /etc/radvd.conf
+#
+{% endblock banner %}
+# Config file for radvd, the router advertisement daemon
+# See man page for radvd.conf for descriptions of all available options
+#
+
+{# If our configuration has VLAN interfaces... #}
+{% if VLAN_INTERFACE %}
+{% for vlan_iface in VLAN_INTERFACE %}
+{# If this VLAN has an IPv6 address... #}
+{% if vlan_iface[1] | ipv6 %}
+interface {{ vlan_iface[0] }}
+{
+    IgnoreIfMissing on;
+    AdvSendAdvert on;
+    MinRtrAdvInterval 200;
+    MaxRtrAdvInterval 600;
+    AdvManagedFlag on;
+    AdvOtherConfigFlag off;
+    AdvLinkMTU 9100;
```

Comment:
apply similiar rule as this commit?

[https://github.com/Azure/sonic-buildimage/commit/a69b353630154935c02db0d287e1a760b2698167](https://github.com/Azure/sonic-buildimage/commit/a69b353630154935c02db0d287e1a760b2698167)

Reply:
In the referenced commit, the MTU is specified in the ConfigDB on a port-by-port basis, and is looked up in the PORT table by port name. However, we do not currently store the MTU length for VLAN interfaces in the ConfigDB. I we decide to store this value at a later point in time, we can modify this accordingly.